nilearn/plotting/displays/__init__.py

```diff
@@ -0,0 +1,34 @@
+"""
+Display objects and utilities.
+
+These objects are returned by plotting functions
+from the :mod:`~nilearn.plotting` module.
+"""
+
+from .axes import BaseAxes, CutAxes, GlassBrainAxes
```

Comment:
I don't think it's important but from comments in other PRs I understood the current trend was to prefer absolute imports?